pkg/framework/framework.go

```diff
@@ -66,7 +75,7 @@ type ContextData struct {
 	// test multiple times in parallel.
 	UniqueName         string
 	CertManagerPresent bool // if crd is detected
-	OperatorMap        map[operators.OperatorType]operators.OperatorSetup
+	OperatorMap        map[operators.OperatorType]operators.OperatorAccessor
```

Comment:
Whi has this been renamed ?

Reply:
Because original intention was to have a configuration class for Operator, `OperatorConfig`. Rename is not needed since the OperatorConfig idea was scrapped, and instead NewFramework accepts pre-configured builders; however OperatorAccessor name is more fitting IMO - since it is a struct that contains information about operator, and methods to teardown operator/namespace.

Reply:
I still prefer Config or Setup, as the objective of the implementation is mainly around setup and teardown of the operator on a given namespace.